unet.py

```diff
@@ -5,13 +5,34 @@
 
 # convolution module
 class ConvModule(nn.Module):
-    def __init__(self, in_channels, out_channels, kernel_size=3, stride=1, padding0=1, padding1=1, final='relu'):
+    def __init__(
+                 self, 
+                 in_channels, 
+                 out_channels, 
+                 kernel_size=3, 
+                 stride=1, 
+                 padding0=1, 
+                 padding1=1, 
+                 final='relu'
+                 ):
         super(ConvModule, self).__init__()
 
         # Convolutions
         # ------------
-        self.conv0 = nn.Conv3d(in_channels, out_channels, kernel_size=kernel_size, stride=stride, padding=padding0)
-        self.conv1 = nn.Conv3d(out_channels, out_channels, kernel_size=kernel_size, stride=stride, padding=padding1)
+        self.conv0 = nn.Conv3d(
+                               in_channels, 
+                               out_channels, 
+                               kernel_size=kernel_size, 
+                               stride=stride, 
+                               padding=padding0
+                               )
+        self.conv1 = nn.Conv3d(
+                               out_channels, 
+                               out_channels, 
+                               kernel_size=kernel_size, 
+                               stride=stride, 
+                               padding=padding1
+                               )
```

Comment:
So, I don't like black, but I like this less. :joy: Can I suggest either black-style formatting:

```python
self.conv1 = nn.Conv3d(
    out_channels, 
    out_channels, 
    kernel_size=kernel_size, 
    stride=stride, 
    padding=padding1,
)
```

Or my own concoction, which uses two indents for the argument list, to visually distinguish line continuations from actual indented blocks:

```python
self.conv1 = nn.Conv3d(
        out_channels, 
        out_channels, 
        kernel_size=kernel_size, 
        stride=stride, 
        padding=padding1,
        )
```

Note that in both cases the last argument has a comma. Also I'm personally completely undecided about the placement of the closing parenthesis. =P PEP8 does not make a prescription either way.

One good reason to *not* align code to the opening parenthesis is that then you need to change all the lines if you change the name of the call, or the name of the variable you're assigning to — which results in unnecessary noise in a git diff, for example.